.github/workflows/vale.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: Vale
+on:
+  pull_request:
+    paths:
+      - 'guides/common/*.adoc'
+      - '.github/workflows/vale.yml'
+
+jobs:
+  vale:
+    name: linter
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: errata-ai/vale-action@reviewdog
+        with:
+          filter_mode: diff_context
+          vale_flags: "--no-exit --minAlertLevel=error"
+          reporter: github-pr-review
+          fail_on_error: false
```

Comment:
Don't we actually want the check to fail if there's an error? Otherwise they're just warnings, right?

Reply:
"fail_on_error: true" blocks merge. I don't recommend it until this feature has been thoroughly tested. [Documentation](https://redhat-documentation.github.io/vale-at-red-hat/docs/main/user-guide/using-vale-github-action/)

Reply:
Yes, it's true that they are just warnings, but if you make a Vale pass mandatory, a merge might be blocked until someone can add an exception.

Reply:
Or we could make it mandatory and see how it goes....

Reply:
That's what I thought. :-) So I will resolve this comment.

Reply:
> A failing check shows up, but doesn't block merge unless you make it a required check. Just as the linkchecker today doesn't block merges but does clearly show.

I will add an error just to make sure that the Vale documentation is accurate.